exporter/elasticsearchexporter/integrationtest/datareceiver.go

```diff
@@ -137,7 +137,7 @@ func newMockESReceiver(params receiver.CreateSettings, cfg *config, next consume
 
 				if err := next.ConsumeLogs(context.Background(), logs); err != nil {
 					response.HasErrors = true
-					item.Status = http.StatusInternalServerError
+					item.Status = 429
```

Comment:
```suggestion
					item.Status = http.StatusTooManyRequests
```
(a bit more informative)